docker/Dockerfile

```diff
@@ -54,10 +54,10 @@ RUN apk add --no-cache \
     eudev	
 
 # Copy files from previous build stage	
-COPY --from=build-z2m /usr/src/app /usr/src/app	
-
-WORKDIR /usr/src/app
+COPY --chown=node:node --from=build-z2m /usr/src/app /usr/src/app
```

Comment:
why do we need to chown everything?

Reply:
I feel like it may need some seccomp config to talk to the zwave interface be it usb/serial etc if it's not root too?

Reply:
As-is it doesn't work because `config/store` needs to open up some `.json` files as read/write, and all of the files have a 755 mask.  Even with that path mounted as a volume (the parent `/usr/src/app/store` folder has the same mask.

So we'll either need to `chmod` the files at container build time (testing this), or `chown` them to `node`.  Any preference, @chrisns?

I'm still open to working out how to make the UID/GID for the `node` user available to modification at runtime by environment vars, but that will require some additional things added to the container to run some scripts as `root` at startup to modify the user.

Reply:
the issue is that we've not defined the VOLUMES more clearly which may have made this slightly less painful

a startup script to chmod kinda defeats the point since it'll need to run as root.

you could introduce a breaking change that checks and bails out early if it fails with some helpful feedback to fix the permissions or do user mapping, user mapping would be preferable anyway to avoid clashing with other things on the host machine

for the minute you could achieve all of what you're setting out to do with an update to the docs, kube+docker-compose config with what we recommend you run it with e.g.
```bash
export HOST_UID=123 # some random number that won't clash, or use UID 0 if you want to run just the container namespacing but still root host mapping (ugly)
docker run -u $HOST_UID:1000 -v $(pwd)/data:/data
```
or whatever

gonna move this PR to a draft if thats ok? until we've got a clear answer that works